### DISCUSSION TITLE
PHP 8 Support

### DISCUSSION THREAD

**Discussion by madhancock** [php-amqplib/php-amqplib#901] [Q&A] [answered]
Hi guys,

Firstly, thanks for your efforts on this project.

Is there a rough ETA on a PHP 8.0 compatible release? I've checked the milestone for 3.0 but I'm not sure if the remaining two items are required for it to ship as there hasn't been much activity on one of them for some time.

Any news would be great. Happy to help where possible!

**Comment by danielbenzie** [comment]
Same here. Happy to help

**Comment by adoy** [comment, marked as answer]
PHP8.0 support will be added in php-amqplib v3.0. I see that the current tag is 3.0.0-rc2. What is the release cycle so that we can approximativaly know when the official version will be released ?

Thanks a lot :-)

**Comment by masterjus** [comment]
Thank you very mush for the update.

Looks like this thread could be closed

**Comment by ghost** [comment]
PHP8 works great for me in production. Thanks a lot for this!
